### PR TITLE
Support private Docker repositories

### PR DESCRIPTION
**Pull request by elgohr**
Private Docker repositories often need a password for authentication (https://docs.cloudfoundry.org/devguide/deploy-apps/push-docker.html#private-repo).
This pull request adds the required password to the environment.

### CODE REVIEW

**Comment by elgohr** [comment]
Was missing docker-username on push.
Added it.

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ be specified.
   that this will re-deploy over. If this is set the resource will perform a
   zero-downtime deploy.
 * `environment_variables`: *Optional.* It is not necessary to set the variables in [manifest][cf-manifests] if this parameter is set.
+* `docker_password`: *Optional.* This is used to authenticate against a protected docker registry.
```
Comment (vito) [inline README.md]: Shouldn't there be a line here for `docker_username` too?

Comment (elgohr) [reply README.md]: Could have sworn that I already added it...

**Comment by vito** [comment]
Thanks!

**Pull request merged**